### PR TITLE
OSDOCS-2403 RHOAM Addons

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR continues the work that JQ started for the RHOAM updates. Fixes #36014 

JIRA: https://issues.redhat.com/browse/OSDOCS-2403

Orignal PR: https://github.com/openshift/openshift-docs/pull/36014

Preview: 

ROSA - https://deploy-preview-37941--osdocs.netlify.app/openshift-rosa/latest/adding_service_cluster/rosa-available-services.html#aws-cloudwatch_rosa-available-services

OSD - https://deploy-preview-37941--osdocs.netlify.app/openshift-dedicated/latest/adding_service_cluster/available-services.html

### CODE REVIEW

File: modules/aws-cloudwatch.adoc [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+= Amazon CloudWatch
 
-You must first install the `cluster-logging-operator` using the `rosa` CLI before installing the AWS CloudWatch service. The AWS CloudWatch add-on can be installed using either the `rosa` CLI or the OpenShift Cluster Manager (OCM). For more information, see link:https://docs.openshift.com/rosa/logging/rosa-install-logging.html#rosa-install-logging[Installing logging add-on services].
+Amazon CloudWatch forwards logs from {product-title} to the AWS console for viewing. You must first install the ROSA `cluster-logging-operator` using the `rosa` CLI before installing the Amazon CloudWatch service through the OpenShift Cluster Manager (OCM) console.
 
 .Additional resources
-* link:https://docs.aws.amazon.com/cloudwatch/?id=docs_gateway[Amazon CloudWatch] documentation
+
+* See xref:../logging/rosa-install-logging.adoc#rosa-install-logging[Install the logging add-on service] for information about the Amazon CloudWatch log forwarding service.
+* link:https://aws.amazon.com/cloudwatch/[Amazon CloudWatch product information]
```
Comment (mburke5678) [inline modules/aws-cloudwatch.adoc:14]: The first bullet is a sentence. This should also be a sentence for parallelism.

File: modules/osd-rhoam.adoc [hunk 2]
```diff
@@ -1,11 +1,24 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/available-services.adoc
+// * adding_service_cluster/rosa-available-services.adoc
 
 [id="osd-rhoam_{context}"]
-
 = Red Hat OpenShift API Management
 
+The Red Hat OpenShift API Management (OpenShift API Management)  service is available as an add-on to your {product-title} on AWS cluster. OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and also includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.
```
Comment (mburke5678) [inline modules/osd-rhoam.adoc:9]: s/and also/and ?? (seems redundant)

File: modules/osd-rhoam.adoc [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+
+This free OpenShift API Management entitlement tier provides:
```
Comment (mburke5678) [inline modules/osd-rhoam.adoc:11]: Do we need _free_ here? Seems like marketing-speak.

File: modules/osd-rhoam.adoc [hunk 4]
```diff
@@ -1,11 +1,24 @@
...
-The Red Hat OpenShift API Management service is available as an add-on to your {product-title} cluster. Red Hat OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform.
-Red Hat OpenShift API Management includes an implementation of the Red Hat Single Sign-On to secure and protect your API and for API Management and not as a company-wide SSO solution.
+ifdef::openshift-rosa[]
+* Availability to any cluster that meets the resource requirements listed in the Red Hat OpenShift API Management service definition.
+endif::[]
+ifdef::openshift-dedicated[]
+* Availability to any cluster that meets the resource requirements listed in the {product-title} service definition.
+endif::[]
+* Full production-level support.
+* No time limits - this is not a trial.
```
Comment (mburke5678) [inline modules/osd-rhoam.adoc:20]: Same with this bullet. Sounds like marketing

**Comment by mburke5678** [comment]
@EricPonvelle A few nits. Otherwise LGTM. One thing, the assembly has _Red Hat OpenShift Service on AWS ROSA_. This should be _Red Hat OpenShift Service on AWS (ROSA)_, correct?

File: modules/osd-rhoam.adoc (hunk 3 above)
Comment (EricPonvelle) [reply modules/osd-rhoam.adoc:11]: I believe this is actually the purpose of the original BZ. It specifies that there's no costs or limits: https://issues.redhat.com/browse/OSDOCS-2403.

Anyway we can word this better?

File: modules/osd-rhoam.adoc [hunk 5]
```diff
@@ -1,11 +1,24 @@
...
+// * adding_service_cluster/available-services.adoc
+// * adding_service_cluster/rosa-available-services.adoc
 
 [id="osd-rhoam_{context}"]
-
 = Red Hat OpenShift API Management
 
+The Red Hat OpenShift API Management (OpenShift API Management) service is available as an add-on to your {product-title} on AWS cluster. OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.
+
+This OpenShift API Management entitlement provides:
```
Comment (michaelryanpeter) [inline modules/osd-rhoam.adoc:11]: Is _entitlement_ the right word here? SSG is particular about not using _entitlement_ when you mean _subscription_. I do not grasp the finer distinctions, so just a question.

File: modules/osd-rhoam.adoc [hunk 6]
```diff
@@ -1,11 +1,24 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/available-services.adoc
+// * adding_service_cluster/rosa-available-services.adoc
 
 [id="osd-rhoam_{context}"]
-
 = Red Hat OpenShift API Management
 
+The Red Hat OpenShift API Management (OpenShift API Management) service is available as an add-on to your {product-title} on AWS cluster. OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.
```
Comment (michaelryanpeter) [inline modules/osd-rhoam.adoc:9]: > OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.

Nit: Could we split this in two?
"OpenShift API Management is a managed API traffic control and API program management solution. It is based on the 3scale API Management platform and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.

Comment (michaelryanpeter) [reply modules/osd-rhoam.adoc:9]: > and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.

Does the following change the meaning too much? "and implements single sign-on for Red Hat solutions." I think this is the preferred usage based on the offical RH products and solutions names list.

**Review by michaelryanpeter — COMMENTED** [review]
Just a couple of questions on specific usage and terminology based on the product list, ISG, and SSG. You might have already considered these, but I just wanted to check.

**Pull request merged**